skyportal/facility_apis/fake_request.py

```diff
@@ -0,0 +1,33 @@
+from tornado import httputil
+
+
+class FakeApplication(object):
```

Comment:
Missing docstring ... also this code style looks different than elsewhere. Is this a copy of code from elsewhere? Should reference the origin if so.

Reply:
The docstring now mentions tornado.web directly.